media/Config/index.js

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const oneImport = {
+  type: "one-import",
+  use: true,
+  options: [
+    { optionName: "bcq", optionValue: false },
+    { optionName: "onnx", optionValue: false },
+    { optionName: "tf", optionValue: false },
+    { optionName: "tflite", optionValue: false },
+  ],
+};
+
+const oneImportBcq = {
+  type: "one-import-bcq",
+  use: false,
+  options: [
+    {
+      optionName: "converter_version",
+      optionValue: "v1",
+      optionType: ["v1", "v2"],
+    },
+    { optionName: "input_path", optionValue: "" },
+    { optionName: "output_path", optionValue: "" },
+    { optionName: "input_arrays", optionValue: "" },
+    { optionName: "input_shapes", optionValue: "" },
+    { optionName: "output_arrays", optionValue: "" },
+  ],
+};
+
+const oneImportOnnx = {
+  type: "one-import-onnx",
+  use: false,
+  options: [
+    { optionName: "input_path", optionValue: "" },
+    { optionName: "output_path", optionValue: "" },
+    { optionName: "input_arrays", optionValue: "" },
+    { optionName: "output_arrays", optionValue: "" },
+    { optionName: "model_format", optionValue: "" },
+    { optionName: "converter_verstion", optionValue: "" },
+  ],
+};
+
+const oneImportTf = {
+  type: "one-import-tf",
+  use: false,
+  options: [
+    {
+      optionName: "converter_version",
+      optionValue: "v1",
+      optionType: ["v1", "v2"],
+    },
+    {
+      optionName: "model_format",
+      optionValue: "graph_def",
+      optionType: ["graph_def", "saved_model", "keras_model"],
+    },
+    { optionName: "input_path", optionValue: "" },
+    { optionName: "output_path", optionValue: "" },
+    { optionName: "input_arrays", optionValue: "" },
+    { optionName: "output_arrays", optionValue: "" },
+    { optionName: "input_shapes", optionValue: "" },
+  ],
+};
+
+const oneImportTflite = {
+  type: "one-import-tflite",
+  use: false,
+  options: [
+    { optionName: "input_path", optionValue: "" },
+    { optionName: "output_path", optionValue: "" },
+  ],
+};
+
+const oneImportOptions = [
+  oneImportBcq,
+  oneImportOnnx,
+  oneImportTf,
+  oneImportTflite,
+];
+
+const optimize = {
```

Comment:
for naming convention, `one-import` uses `oneImport` but this uses `optimize` not `oneOptimize`
it would be better to go with one convention, either is OK for me.

Reply:
okay. I changed optimize to oneOptimize